src/string.rs

```diff
@@ -125,8 +125,12 @@ impl Ord for IString {
         Ord::cmp(AsRef::<str>::as_ref(self), AsRef::<str>::as_ref(other))
     }
 }
+impl PartialOrd for IString {
```

Comment:
a description for this change should probably also be reflected in a code comment

Reply:
Done